python/ray/dataframe/dataframe.py

```diff
@@ -1222,7 +1228,9 @@ def __iter__(self):
         raise NotImplementedError("Not Yet implemented.")
 
     def __contains__(self, key):
-        raise NotImplementedError("Not Yet implemented.")
+        return any(
```

Comment:
This will simplify to `key in self.columns`, but will end up being slower with `ray.get()`.

Reply:
resolved by calling key in self.columns